server/app/fixtures.py

```diff
@@ -133,9 +106,20 @@ def create_payments(tasks, users):
 
 # Populating Mock data
 def create_mock_data(sender, **kwargs):
-    delete_mock_data()
+    # delete_mock_data()
     users = create_users()
-    create_profiles(users)
     tasks = create_tasks(users)
     create_bids(tasks, users)
     create_payments(tasks, users)
+
+
+# Empty database before populating mock data
+# def delete_mock_data():
+#     if Users.objects.exists():
+#         Users.objects.all().delete()
+#         Tasks.objects.all().delete()
+#         Bids.objects.all().delete()
+#         Payments.objects.all().delete()
+#         print("Mock data deleted.")
+#     else:
+#         print("No mock data to delete.")
```

Comment:
remove, or uncomment if this code is used

Reply:
ok boss xd